pom.xml

```diff
@@ -196,7 +196,11 @@
     <datanucleus-core.version>4.1.17</datanucleus-core.version>
     <guava.version>14.0.1</guava.version>
     <janino.version>3.1.9</janino.version>
-    <jersey.version>2.36</jersey.version>
+    <!--
```

Comment:
jersey 3.x only support Java 11+, maybe we can upgrade to 3.x in Spark 4.0 :)

Reply:
Good proposal!

Reply:
+1 for that.